src/Orchard.Web/Core/Common/Views/CommonMetadataOwner.cshtml

```diff
@@ -11,4 +11,4 @@
     // owner isn't really who last modified this, is it?
     IUser owner = commonPart.As<CommonPart>() == null ? null : commonPart.As<CommonPart>().Owner;
 }
-@T("By {0}", owner == null ? T("unknown").ToString() : Convert.ToString(Html.ItemDisplayText(owner)))
\ No newline at end of file
+@T("By {0}", owner == null ? T("unknown").ToString() : Convert.ToString(Html.ItemDisplayText(owner, false)))
```

Comment:
Your proposol is not working here. I am getting:

CS0173: Type of conditional expression cannot be determined because there is no implicit conversion between 'Orchard.Localization.LocalizedString' and'System.Web.Mvc.MvcHtmlString'